src/mailkit.cs

```diff
@@ -0,0 +1,423 @@
+using System;
+using CoreGraphics;
+using Foundation;
+using ObjCRuntime;
+using AVFoundation;
+using AppKit;
+
+namespace MailKit {
+	[NoWatch, NoTV, NoiOS, NoMacCatalyst, Mac (12,0)]
+	[Native, Advice ("'MEComposeSessionErrorCode' is not available in UIKit on macOS.")]
```

Comment:
You can removed all `[Advice ("'MEComposeSessionErrorCode' is not available in UIKit on macOS.")]`
That was needed before we had `[NoMacCatalyst]` attributes